docs/farming-&-staking/wallets/polkadot.md

```diff
@@ -94,7 +94,7 @@ If you face any trouble or would like to learn about other features for Polkadot
 
     ![trouble-3](/img/doc-imgs/polkadot/trouble-3.png)
 
-1. Go to the Subspace/Polkadot Explorer here: [Polkadot/Substrate Portal](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftest-rpc.subspace.network#/explorer)
+1. Go to the Subspace/Polkadot Explorer here: [Polkadot/Substrate Portal](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.taurus.subspace.foundation%2Fws#/explorer)
```

Comment:
Let's put the mainnet explorer instead of the taurus in this case :) 

```suggestion
1. Go to the Subspace/Polkadot Explorer here: [Polkadot/Substrate Portal](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.mainnet.subspace.foundation%2Fws#/explorer)
```